src/Docs/Router.fs

```diff
@@ -14,31 +14,35 @@ type Page =
     | TooltipInstallation
     | CheckradioOverview
     | CheckradioInstallation
+    | SwitchOverview
+    | SwitchInstallation
     | PopoverOverview
     | PopoverInstallation
     | PageLoaderOverview
     | PageLoaderInstallation
 
 let defaultPage = BulmaOverview
-    
+
 let parseUrl = function
-    | [ "" ] -> BulmaOverview 
-    | [ "installation" ] -> BulmaInstallation 
-    | [ "api-description" ] -> BulmaAPIDescription 
-    | [ "quickview"; "installation" ] -> QuickViewInstallation 
+    | [ "" ] -> BulmaOverview
+    | [ "installation" ] -> BulmaInstallation
+    | [ "api-description" ] -> BulmaAPIDescription
+    | [ "quickview"; "installation" ] -> QuickViewInstallation
     | [ "quickview" ] -> QuickViewOverview
-    | [ "calendar"; "installation" ] -> CalendarInstallation 
```

Comment:
I'm sorry about this, I disabled fantomas after a few changes and forgot to revert a couple of files, if you want I can revert these changes

Reply:
No problem at all 😄